### ISSUE TITLE
'npm install selenium-webdriver' failed with iojs 3.0.0

### ISSUE THREAD

**Issue by Dzenly** [SeleniumHQ/selenium#904]
I created an issue here:
https://github.com/nodejs/io.js/issues/2337

**Comment by lukeis** [comment]
can you please leave a better description here? like how to reproduce your issue (example steps of what you're doing to 'build')

**Comment by Dzenly** [comment]
Ok, I changed the subject to be crystal clear. Sorry for such garbage (I am beginner here).
In fact guys from iojs already understood the problem and made pull request to according modules which selenium-webdriver depends on (they uses old nan module). Though it is not iojs problem.

**Comment by jleyba** [comment]
Iojs is not an officially supported platform. The `selenium-webdriver` npm module supports the last two stable releases for node: 0.10.x & 0.12.x. iojs is like node 0.11/0.13: if it works, great. If not - sorry.

That said, I'd be happy to review any PRs that address iojs issues. For now, I'm closing this because there's no action required by the Selenium project.

**Comment by Dzenly** [comment]
You'd be happy to review PR from iojs, but with sorry :) Okaay :)

**Comment by jleyba** [comment]
I guess it is a weird position: "we'll accept your PR, but won't make promises about it working long term."

Let me try again:

As the sole maintainer of Selenium's client library for node, I don't consider iojs a supported platform. My time with Selenium is limited these days and it would not be prudent for me to extend support to a new platform. Community contributions to support iojs along with node are welcome and will not be rejected on the grounds of iojs being "unsupported". Unfortunately, until more members of the Selenium project start working on the JS client, _official_ support will remain focused on node and regressions may crop up.

Having a single project member dedicated to one of the client language bindings is not ideal, but that's just how things have worked out. Members tend to focus on maintaining bindings for the languages they work in.

The __TL;DR__ is still the same as before, but hopefully my position is easier to understand.

**Comment by Dzenly** [comment]
Understood. I didn't know that you are alone on this project. Great thank you for your work!

**Comment by lseongjoo** [comment]
npm install fails with io.js 3.0.0 on OS X too.

**Comment by Dzenly** [comment]
Now it is absolutely selenium-webdriver problem. Due to using "ws": "^0.7.1", package.json.
What if you will change it to ">=0.7.1"?

**Comment by Dzenly** [comment]
Okaay :) Is new nodejs 4.0.0 "officially supported platform" ?
